pkg/model/config/config.go

```diff
@@ -119,6 +119,17 @@ func (c Config) HasGroup(group string) bool {
 	return false
 }
 
+// HasWebhook returns true if webhook is already present
+func (c Config) HasWebhook(kind string) bool {
+	for _, r := range c.Resources {
+		if r.Kind == kind {
```

Comment:
we need to compare here gkv becuase we can have the same Kind for diff apis/gkv. 
/hold